### DISCUSSION TITLE
insert the possibility of programmer by seconds

### DISCUSSION THREAD

**Discussion by rubenwilson1** [AzuraCast/AzuraCast#4910] [General]
good afternoon is possible next
version to be able to program by seconds the programming? I comment on it because there are certain moments in which certain events I want that act hour, minute and second act to enter.
could it be possible that it could be done? Thank you.

**Comment by rubenwilson1** [comment]
hello?

**Reply by BusterNeece** [reply]
Don't post things like this. We're a tiny team building a free piece of software; we'll get back to your request as soon as possible.

**Comment by rubenwilson1** [comment]
I'm sorry, you're right, but it's been days since I published the original post and just as you answered me to tell me this, you could take advantage of and answer the question I asked you, that the contribution to improving the software is what I intend to give you from my humble wisdom as radio director, these are things that I think are needed and that would be fine
If I have made that second post, it is to give life to a post that has been there for 4 days and that if I neglect it, it happens like the one from likysoapp that it took you a month or more to respond. I apologize but I would like my suggestions to be taken into account. Thank you.

**Reply by BusterNeece** [reply]
For one, it is not appropriate that when I ask you to stop posting things like "Hello?" you further criticize our response times. We do not owe you anything in terms of support; we provide a huge, complex product free of charge and work on it every single day for years now, and any support we can provide is an _extra_ on top of that.

Second, what you're requesting is a feature, and should instead be posted over at https://features.azuracast.com/ if you haven't already.

**Comment by rubenwilson1** [comment]
You are absolutely right, constructive criticism is not a criticism to hurt, excuse me but I would not like you to see it that way, I know it is an extra and that perhaps you are not gaining anything with this, but I also thank you for answer, because I think that is worth a lot I have apologized for the previous message, I am going to post in that url that you have given me, well, I had not done it, again apologies.
